spikingjelly/activation_based/triton_kernel/flexsn/template.py

```
@@ -199,13 +250,91 @@ def get_flexsn_inference_kernel(
     return kernel_exe
 
 
+def get_flexsn_inference_final_state_kernel(
+    core_str: str, core_name: str, info: FlexSNInfo, verbose: bool = False
+):
+    hash = f"{core_name[-8:]}_{INFERENCE_FINAL_STATE_AUTOTUNE_TAG}"
```

Ruff (0.15.11)

[error] Variable `hash` is shadowing a Python builtin

(A001)